### PR TITLE
Sloeber does not install in Eclipse 2022-12 #1529

### PR DESCRIPTION
**Pull request by wimjongman**
Updated the update sites in the products
Updated the update sites in the features

### CODE REVIEW

File: io.sloeber.feature/feature.xml [hunk 1]
```diff
@@ -32,6 +32,11 @@ Trump211
 If you do not agree with this policy, please do not install this software.
    </license>
 
+   <url>
+      <discovery label="Eclipse Nebula Widgets" url="https://download.eclipse.org/nebula/releases/latest/"/>
+      <discovery label="Eclipse Nebula Incubation Widgets" url="https://download.eclipse.org/nebula/incubation/latest"/>
```
Comment (jantje) [inline io.sloeber.feature/feature.xml:37]: Do we still want to add this?
This has been added after you did some changes to the plotter Sloeber needed but the plotter has been released and stable for a long time now.

File: io.sloeber.product.sdk/arduino.product [hunk 2]
```diff
@@ -280,14 +280,15 @@ United States, other countries, or both.
...
-      <repository location="http://gnu-mcu-eclipse.netlify.com/v4-neon-updates/" enabled="true" />
-      <repository location="http://download.eclipse.org/releases/latest/" enabled="true" />
+      <repository location="https://eclipse.baeyens.it/update/V4/nightly/" enabled="false" />
+      <repository location="https://eclipse.baeyens.it/update/V4/stable/" enabled="true" />
+      <repository location="https://download.eclipse.org/releases/latest/" enabled="true" />
       <repository location="https://download.eclipse.org/technology/babel/update-site/latest/" enabled="true" />
-      <repository location="http://https://download.eclipse.org/justj/jres/15/updates/release/latest/" enabled="true" />
       <repository location="https://download.eclipse.org/eclipse/updates/latest" enabled="true" />
+      <repository location="https://download.eclipse.org/justj/jres/17/updates/release/latest/" enabled="true" />
+      <repository location="https://download.eclipse.org/nebula/incubation/latest" enabled="true" />
```
Comment (jantje) [inline io.sloeber.product.sdk/arduino.product:289]: Should this not be removed or at least be false?

File: io.sloeber.feature/feature.xml (hunk 1 above)
Comment (wimjongman) [reply io.sloeber.feature/feature.xml:37]: Yes. When do not install from the product this will add the required nebula repos.

Comment (wimjongman) [reply io.sloeber.feature/feature.xml:37]: I also updated the target file here to include 0.0.0 versions

File: io.sloeber.product.sdk/arduino.product (hunk 2 above)
Comment (wimjongman) [reply io.sloeber.product.sdk/arduino.product:289]: Yes, I removed nebula incubation. Will also remove it from the feature.

File: io.sloeber.feature/feature.xml (hunk 1 above)
Comment (jantje) [reply io.sloeber.feature/feature.xml]: Thanks.
It was on my todo list.

**Comment by wimjongman** [comment]
Abandonded

**Pull request closed without merging**